gh-87063: Add `closed` attribute to multiprocessing.Process (#125838)
python/cpython: Doc/whatsnew/3.14.rst

(Contributed by Trey Hunner in :gh:`122873`.)

multiprocessing
---------------

* Add the :attr:`~multiprocessing.Process.closed` property to
  :class:`multiprocessing.Process` objects to determine whether
  they have been closed or not.

> 

Reviewer: You should re-add the "Contributed by [...]" part that you had. It was correct before.

  For more details, please see :ref:`pickle protocols <pickle-protocols>`.

> multiprocessing

Reviewer: Please, order the module's entry alphabetically. This section should be placed between the `json` and `operator` sections above.

---------------

> * Add attribute :attr:`!closed` to :class:`multiprocessing.Process`,
>   It is used to check whether the process is closed.

Reviewer: ```suggestion
* Add the :attr:`~multiprocessing.Process.closed` property to
  :class:`multiprocessing.Process` objects to determine whether
  they have been closed or not.
```
